lib/miq_automation_engine/models/miq_ae_domain.rb

```diff
@@ -49,6 +49,19 @@ def unlock_contents!
     save!
   end
 
+  def contents_locked?
+    system
+  end
+
+  def editable_properties?
+    name != 'ManageIQ'
```

Comment:
@Fryguy 
This is temporary till we implement the source=system|remote|user|user_locked